scripts/travis/travis-tests

```diff
@@ -19,6 +19,27 @@ if [ "$UNAME_M" != "x86_64" ]; then
 	SKIP_TRAVIS_TEST=1
 fi
 
+install_retry_counter=0
+max_apt_retries=5
+
+# This function loops a couple of times over apt-get, hoping to
+# avoid CI errors due to errors during apt-get
+# hashsum mismatches, DNS errors and similar things
+apt_install () {
+	while true; do
+		let install_retry_counter+=1
+		if [ $install_retry_counter -gt $max_apt_retries ]; then
+			exit 1
+		fi
+		apt-get clean -qqy || continue
+		apt-get update -qqy || continue
+		apt-get install -qqy --no-install-recommends $@ && break
+
+		# In case it is a network error let's wait a bit.
+		sleep $install_retry_counter
+	done
```

Comment:
@adrianreber Hi, shouldn't we reset install_retry_counter to 0 at the end?

Reply:
Yes, right. Thanks. I will fix it in a follow up PR. I was planning to move it to its own script and also use it in all the Dockerfiles.